app/models/git/exercise.rb

```diff
@@ -195,7 +200,8 @@ def important_filepaths
         introduction_append_filepath,
         hints_filepath,
         *test_filepaths,
-        *editor_filepaths
+        *editor_filepaths,
+        *invalidator_filepaths
```

Comment:
Including the invalidator filepaths in this list causes them to be included in the important files hash